doc/NPCs.md

```diff
@@ -547,7 +547,7 @@ The NPC will offer you a list of missions for your allied NPCs, depending on the
 
 #### u_add_effect: effect_string, (optional duration: duration_string)
 #### npc_add_effect: effect_string, (optional duration: duration_string)
```

Comment:
I don't believe this is actually optional, unless you have changed that. Is there an 'else' case for if the string isn't specified?

If not, it would be reasonable to make it permanent if not specified, but then we should clarify that in the documentation. If it's made that way, it will shorten quite a few files in the future.

Reply:
Adjusted the documentation because as you say, duration really isn't optional.